storybook/stories/chart/piechart/PieChart.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Pie, PieChart, ResponsiveContainer } from '../../../../src';
+import { pageData } from '../../data';
+
+export default {
+  title: 'Recharts/Chart/Pie Chart',
+  component: PieChart,
+};
+
+export const Simple = {
+  render: (args: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <PieChart width={800} height={400} startAngle={180} endAngle={0} outerRadius={80}>
+          <Pie data={args.data} dataKey="uv" />
```

Comment:
Interesting, why do we pass the data to the Pie instead of to the Chart? 
I wonder which case we should suggest as the default / recommend / basic usage?

Reply:
Consistent we the other basic stories, lets rather pass it to the PieChart. I think passing the data to the Pie should be an advanced story - because it begs the question what happens if we pass different data to different Pies

Reply:
Thanks for your review.
I will reflect what you said in the next stories.

Reply:
Sorry, I was wrong. The interface is different on the PieChart, and it does not handle the data itself. 🤔

Reply:
Fixed it in https://github.com/recharts/recharts/pull/3329